labs-ios-starter/Storyboards/Map/MapViewController.swift

```diff
@@ -72,11 +72,40 @@ extension MapViewController: CLLocationManagerDelegate {
     }
 }
 
+extension MapViewController: MKMapViewDelegate {
+    func mapView(_ mapView: MKMapView, viewFor annotation: MKAnnotation) -> MKAnnotationView? {
+        let identifier = "Pin"
+        var annotationView = mapView.dequeueReusableAnnotationView(withIdentifier: identifier)
+        if annotationView == nil {
+            annotationView = MKPinAnnotationView(annotation: annotation, reuseIdentifier: identifier)
+            annotationView?.canShowCallout = true
+            annotationView?.rightCalloutAccessoryView = UIButton(type: .detailDisclosure)
+        } else {
+            annotationView?.annotation = annotation
+        }
+        return annotationView
+    }
+    
+    func mapView(_ mapView: MKMapView, annotationView view: MKAnnotationView, calloutAccessoryControlTapped control: UIControl) {
+        let ac = UIAlertController(title: locationName!, message: "What would you like to do?", preferredStyle: .alert)
+        ac.addAction(UIAlertAction(title: "Add To Favorites", style: .default, handler: { (_) in
+            //Handle adding to favorites
+        }))
+        ac.addAction(UIAlertAction(title: "Details", style: .default, handler: { (_) in
+            //Handle viewing details
```

Comment:
I like the comments as reminders for what needs to be done once the backend is connected. Should these be TODO's?

Reply:
Yes the could be marked as TODO, do you think it's a necessary change? :)

Reply:
I think it would be beneficial, yes. :-)